teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -1,5 +1,7 @@
 # Guide to Being a Manager
 
+**Italics indicate sections that still should be reviewed.** 
+
 Reviewers are in charge of **curriculum teams.** Each team consists of 4-6 people who work on different modules of curriculum. They will be tasked with a project, usually covering a topic, and a deadline. By this deadline, all activities, labs and workshops for that project should be completed.
```

Comment:
Are "managers" and "reviewers" being used interchangeably or should we stick to one for consistency?

Reply:
"managers" should be used